servers/camera_server.h

```diff
@@ -59,6 +59,7 @@ class CameraServer : public Object {
 	};
 
 	typedef CameraServer *(*CreateFunc)();
+	static inline constexpr const char feeds_updated_signal_name[] = "feeds_updated";
```

Comment:
Can you update this to `camera_feeds_updated` to keep consistent with the other signal naming convention.

Reply:
done